src/common/column_matrix.h

```diff
@@ -86,9 +86,9 @@ class ColumnMatrix {
     type_.resize(nfeature);
     std::fill(feature_counts_.begin(), feature_counts_.end(), 0);
 
-    uint32_t max_val = 0;
+    uint64_t max_val = 0;
```

Comment:
This section of code operates on `HistCutMatrix`, which stores the quantiles. So `uint32_t` is fine here.